jumpscale/packages/vdc_dashboard/chats/extend_kubernetes.py

```diff
@@ -17,7 +17,7 @@ def flavor(self):
         self.vdc_name = list(j.sals.vdc.list_all())[0]
         self.user_info_data = self.user_info()
         self.username = self.user_info_data["username"]
```

Comment:
I think these 2 lines are not needed in this case.

Reply:
yes, you are right